users_app/pipeline/pipeline.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.models import Group
+
+from users_app import ALLOWED_AP_ACCOUNTS
+
+
+def add_user_to_group(is_new, user, *args, **kwargs):
```

Comment:
Do you mean something like this?
if is_new: 
    if user.email in ALLOWED_AP_ACCOUNTS:
        ....
    elif user.email.endswith('@eventbrite.com'):
        ...
    else:
        ...

Reply:
I think we should change it because people is going to signup just one time, and they will be assigned to a group just once, so "is_new" is going to be True just once. (then they sign in, so the "is_new" is going to be False) 
Also "is_new" is going to be evaluated only one time instead of evaluated three times.